MRN-Code/quarterback
# Add joi schema to sites controller

* **Problem:** The `/sites` route has no documented return value
* **Solution:** Add a joi schema!
* **Testing:**
  1. Spin up a server
  2. Run some requests through the _/swagger/documentation_ site getter
  3. Confirm joi doesn't explode
